Content.Shared/Implants/SharedSubdermalImplantSystem.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Linq;
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Implants.Components;
+using Content.Shared.Tag;
+using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Implants;
+
+public abstract class SharedSubdermalImplantSystem : EntitySystem
+{
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly SharedActionsSystem _actionsSystem = default!;
+    [Dependency] private readonly SharedContainerSystem _container = default!;
+    [Dependency] private readonly TagSystem _tag = default!;
+
+    public const string ImplantSlotId = "implantcontainer";
```

Comment:
```suggestion
    public const string ImplantSlotId = "implant";
```

container is redundant, my pet peeve